configure.ac

```diff
@@ -116,8 +116,28 @@ AC_ARG_ENABLE([test],
   AS_HELP_STRING([--disable-test], [test suite]),
   [], [enable_test=yes])
 
+AC_ARG_WITH([seccomp],
+  AS_HELP_STRING([--with-seccomp=(yes|no)],
+                 [support for --seccomp]))
+AS_CASE([$with_seccomp],
+  [yes],  # explicit “yes”
+    [want_seccomp=yes
+     need_seccomp=yes
+     msg_seccomp=yes],
+  [no],   # explicit “no”
+    [want_seccomp=no
+     need_seccomp=no
+     msg_seccomp=no],
+  [''],   # option not specified
+    [want_seccomp=yes
+     need_seccomp=no
+     msg_seccomp='if tested working'],
+  [*],    # anything else
+    [AC_MSG_ERROR([invalid --with-seccomp arg: $with_seccomp])])
+
 AC_ARG_WITH([libsquashfuse],
```

Comment:
The patch hunk below for the `--with-libsquashfuse` help string modernization doesn't seem to be related to this overall PR; should it be moved to a different one, or did I overlook something?

Reply:
Yeah, that unrelated fix is a little sloppy. ;)